README.md

```diff
@@ -4,14 +4,32 @@ Zenith substitutes PostgreSQL storage layer and redistributes data across a clus
 
 ## Running local installation
 
-1. Build zenith and patched postgres
+1. Install build dependencies and other useful packages
+
+On Ubuntu this set of packages should be sufficient to build the code:
+```text
+build-essential libreadline-dev zlib1g-dev flex bison libxml2-dev
+libcurl4-openssl-dev libssl-dev clang
+```
+
+[Rust] 1.48 or later is also required.
+
+To run the `psql` client, install the `postgresql-client` package or modify `PATH` and `LD_LIBRARY_PATH` to include `tmp_install/bin` and `tmp_install/lib`, respectively.
```

Comment:
Off-topic: I think it would be handy to provide cli command for psql to avoid entering different port each time you want to connect. Something like `zenith pg psql main` or `zenith psql main`. What do you think?

Reply:
How often do people use `psql` as their client? I wonder how useful this really is, because it can't extend to any other clients (or libraries).

Is there a common convention for specifying host+port in an environment variable? We would only be able to supply one at a time, though.

Reply:
Quite often: https://stackoverflow.com/questions/9463318/how-to-exit-from-postgresql-command-line-utility-psql =)